### PR TITLE
docs(examples): add react 18 example about problem of suspense prerendering siblings

### PR DESCRIPTION
**Pull request by manudeli**
Thanks to @minsour @sonsurim @2-NOW. I added you guys as co-author

### CODE REVIEW

File: examples/vite-react-18-suspense-waterfall/index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```
Comment (gwansikk) [inline examples/vite-react-18-suspense-waterfall/index.html:6]: How about changing the `<title>` if user can access it?

**Review by gwansikk — APPROVED** [review]
Everything is perfect! 👍👍
@minsour @sonsurim @2-NOW Thank you!

File: examples/vite-react-18-suspense-waterfall/index.html (hunk 1 above)
Comment (manudeli) [reply examples/vite-react-18-suspense-waterfall/index.html:6]: Could you update this branch as you want please? 🙇

Comment (gwansikk) [reply examples/vite-react-18-suspense-waterfall/index.html:6]:
```suggestion
    <title>Suspensive - Example Waterfall with Suspense</title>
```
🤔 How about this??

Comment (manudeli) [reply examples/vite-react-18-suspense-waterfall/index.html:6]: Cool!

**Comment by manudeli** [comment]
@gwansikk @minsour @sonsurim @2-NOW I renamed this as "Problem of suspense prerendering siblings"

**Pull request merged**